openstudiocore/ruby/openstudio/sketchup_plugin/OpenStudio.rb

```diff
@@ -20,20 +20,22 @@
 require 'extensions.rb'   # defines the SketchupExtension class
 
 # look for the new plugin
-new_plugin = Sketchup.find_support_file("OpenStudio-config", "Plugins/OpenStudio")
+# This doesn't work anymore. Consider using openstudio -v from the cli
+#new_plugin = Sketchup.find_support_file("OpenStudio-config", "Plugins/OpenStudio")
+new_plugin = true
```

Comment:
aside from a link from the plugins directory to the os install tree, i have tested this from install and it works.